.travis/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+tag_version=$1
+
+mkdir ~/.bintray/
+file=$HOME/.bintray/.credentials
+cat <<EOF >$file
+realm = Bintray API Realm
+host = api.bintray.com
+user = $BINTRAY_SNOWPLOW_MAVEN_USER
+password = $BINTRAY_SNOWPLOW_MAVEN_API_KEY
+EOF
+
+cd $TRAVIS_BUILD_DIR
+
+project_version=$(sbt version -Dsbt.log.noformat=true | perl -ne 'print "$1\n" if /(\d+\.\d+\.\d+[^\r\n]*)/' | head -n 1 | tr -d '\n')
```

Comment:
we have found this to be failing in scala-weather if there are dependency conflicts, we fixed it by replacing `head` with `tail`